decentraland/explorer-desktop
# feat: added new graphics quality preset & windowMode set to Borderless

## What does this PR change?

- Settings get initialized from `MainDesktop`
- Replaced default graphics quality preset path
- Changed default window mode to `Borderless`

## How to test the changes?

1. Go to: https://play.decentraland.org/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:feat/desktop-default-settings
2. Open settings menu
3. Change graphics settings
4. Hit the `Reset All` button
5. Preset should be at `Ultra`

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
